serverless/installing-service-mesh-control-plane.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// serverless/installing-openshift-serverless.adoc
+
+[id="installing-service-mesh-control-plane_{context}"]
+= Installing the ServiceMeshControlPlane
+include::modules/common-attributes.adoc[]
+:context: installing-service-mesh-control-plane
+
+Service Mesh is comprised of a data plane and a control plane. After you install the ServiceMesh operator, you can install the control plane. The control plane manages and configures the sidecar proxies to enforce policies and collect telemetry. The following procedure installs a version of the ServiceMesh control plane that acts as an ingress to your applications.
+
+[NOTE] 
+====
+You must install the control plane into the `istio-system` namespace. Other namespaces are currently not supported.
+====
+
+[source,yaml]
+.Sample servicemesh-installation.yaml
+----
+apiVersion: maistra.io/v1
+kind: ServiceMeshControlPlane
+metadata:
+ name: basic-install
+ namespace: istio-system <1>
+spec:
+ istio:
+   global:
+     multitenant: true
+     proxy:
+       autoInject: disabled
+     omitSidecarInjectorConfigMap: true
+     disablePolicyChecks: false
+   istio_cni:
+     enabled: true
+   gateways:
+     istio-ingressgateway:
+       autoscaleEnabled: false
+       type: LoadBalancer
+     istio-egressgateway:
+       enabled: false
+     cluster-local-gateway:
+       autoscaleEnabled: false <1>
+       enabled: true
+       labels:
+         app: cluster-local-gateway
+         istio: cluster-local-gateway
+       ports:
+         - name: status-port
+           port: 15020
+         - name: http2
+           port: 80
+           targetPort: 8080
+         - name: https
+           port: 443
+   mixer:
+     enabled: false
+     policy:
+       enabled: false
+     telemetry:
+       enabled: falsely 
+   pilot:
+     autoscaleEnabled: false
+     sidecar: false
+   kiali:
+     enabled: false nhj
+   tracing:
+     enabled: false
+   prometheus:
+     enabled: false
+   grafana:
+     enabled: false
+   sidecarInjectorWebhook:
+     enabled: false
+----
+<1> Autoscaling is disabled in this version. This release is not intended for production use.
+
+[NOTE]
+====
+Running ServiceMesh with a sidecar injection enabled with OpenShift Serverless is currently not recommended. 
```

Comment:
Just a note: The service mesh team has a fix for the blocker issue that prevented us from running with sidecar injection. We might need to revisit what we want to use when the fix lands and testing shows that it works.